README.md

```diff
@@ -9,7 +10,7 @@
 ## Active proposals
 
 Proposals follow [this process document](https://tc39.github.io/process-document/).
-This list contains only stage 1 proposals and higher that have not yet been withdrawn/rejected, or become finished.
+This list contains only stage 2 proposals and higher that have not yet been withdrawn/rejected, or become finished.
```

Comment:
Would this be a good place to give a quick summary of what this level means? E.g.,

> The committee expects these features to be developed and eventually included in the standard

Reply:
We could do that, sure - the process document covers that tho already?

Reply:
Right, but this may be an important point to highlight in order to keep people from getting confused. The wording is cribbed from the process doc, so it wouldn't be inventing a new definition.